app/src/main/java/com/xmartlabs/template/module/GsonModule.java

```diff
@@ -20,6 +24,18 @@
   @Singleton
   public Gson provideGson() {
     GsonBuilder gsonBuilder = new GsonBuilder()
+        .setExclusionStrategies(new ExclusionStrategy() {
```

Comment:
I think this is not well indented. This line should have long indentation (as being a continuation of the last one), and the line above this should have one more indentation than this one.